mount.go

```diff
@@ -96,6 +76,68 @@ func Mount(
 		err = fmt.Errorf("mount (background): %v", err)
 		return
 	}
+	return
+}
+
+// FuseMount attempts to mount a file system on the given directory. It can block
+// fuse server setup completion using supplied ready channel.
+func FuseMount(
+	dir string,
+	config *MountConfig,
+	ready chan error) (dev uintptr, err error) {
```

Comment:
These use of this channel isn't clear to me. Please elaborate in the documentation what it's for, and when an error may be delivered on the channel versus returned from the function. Please also mark it as a send-only channel to make the intention clear.

Reply:
Okay I'll add a comment above the func definition, explaining what it's for + when error delivered.

> Please also mark it as a send-only channel to make the intention clear.

I'll push a commit later. Thanks.